packs/equipment-effects/effect-ghostcallers-planchette-armor.json

```diff
@@ -21,13 +21,61 @@
             "title": "Pathfinder Treasure Vault"
         },
         "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "predicate": [
+                            {
+                                "or": [
+                                    "parent:origin:item:slug:ghostcallers-planchette",
+                                    {
+                                        "not": "parent:origin:item"
+                                    }
+                                ]
+                            }
+                        ],
+                        "value": "Compendium.pf2e.equipment-srd.Item.Ghostcaller's Planchette"
+                    },
+                    {
+                        "predicate": [
+                            {
+                                "or": [
+                                    "parent:origin:item:slug:ghostcallers-planchette-greater",
+                                    {
+                                        "not": "parent:origin:item"
+                                    }
+                                ]
+                            }
+                        ],
+                        "value": "Compendium.pf2e.equipment-srd.Item.Ghostcaller's Planchette (Greater)"
+                    }
+                ],
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.EquipmentVariety",
+                "rollOption": "ghostcallers-planchette"
+            },
             {
                 "exceptions": [
                     "ghost-touch"
                 ],
                 "key": "Resistance",
+                "predicate": [
```

Comment:
I wonder if we should have a custom exception to handle "attacks by incorporeal creatures count as ghost touch"?

Reply:
Probably- how do we set that up?

Reply:
Take a look at Vampire Count. `PF2E.IWR.Custom.MagicalSilver`